.github/workflows/namespace-notifier.yml

```diff
@@ -0,0 +1,26 @@
+name: Buildpack Registry Namespace Notifier
+on: 
+  push:
+    branches:
+      - main-test
+
+jobs:
+  email-maintainers:
+    name: Email Maintainers
+    runs-on: ubuntu-latest
+    steps:
+    - name: Send mail
+      uses: dawidd6/action-send-mail@v2
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{ secrets.MAIL_USERNAME }}
+        password: ${{ secrets.MAIL_PASSWORD }}
+        subject: GITHUB ACTIONS ${{ github.repository }} updated !!!
+        body: Maintainers, there are some changes in ${{ github.repository }} repository.
+        to: ${{ secrets.MAIL_RECEIVER }}
+        from: ${{ github.repository }}
```

Comment:
Is this right, or should it be something like `MAIL_SENDER`?

Reply:
Or maybe `noreply@...`

Reply:
Yeah, other than repo's name sounds good. I have updated it as `secrets.MAIL_SENDER`.